cluster/juju/layers/kubernetes-worker/debug-scripts/kubectl

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+set -ux
+
+sudo -iu ubuntu kubectl cluster-info > $DEBUG_SCRIPT_DIR/cluster-info
```

Comment:
I don't recall why exactly, but kubectl wasn't picking up kubeconfig as root. That problem only occurred within the script, though - microbot etc still worked fine. I'll revisit this.